pages/conduct.html

```diff
@@ -37,9 +38,9 @@
 <ol>
   <li>
     <p>
-      Harassment includes offensive verbal comments related to gender,
+      Harassment includes offensive verbal or written comments related to gender,
       sexual orientation, disability, physical appearance, body size,
-      race, religion, sexual images in public spaces, deliberate
+      race, or religion, sexual images in public spaces, deliberate
```

Comment:
On Mon, May 09, 2016 at 01:27:01AM -0700, Simon Waldman wrote:
> … I get a chance in the next few days, file a second PR that
> improves its readability.

+1 to small, atomic PRs.